ClassySchedule/src/main/java/courses/AddCourseToDatabaseController.java

```diff
@@ -1,6 +1,7 @@
 package courses;
 
 import database.Database;
```

Comment:
Delete unneeded import

Also, if we are not using the Database class at all, please delete that as well.